3rdparty/python/constraints.txt

```diff
@@ -13,9 +13,9 @@ iniconfig==1.1.1
 monotonic==1.5
 mypy==0.800
 mypy-extensions==0.4.3
-packaging==20.8
+packaging==20.9
 pex==2.1.28
-pip==20.2.3
+pip==20.2.1
```

Comment:
This downgrade from an old version to an even older version is unexpected but possible. It should be harmless either way as well.

Reply:
hmm.. not sure why this happened.
.